src/Compiler/Service/IncrementalBuild.fs

```diff
@@ -1358,7 +1368,8 @@ type IncrementalBuilder(initialState: IncrementalBuilderInitialState, state: Inc
         | (ilAssemRef, tcAssemblyDataOpt, tcAssemblyExprOpt, boundModel), timestamp ->
             let cache = TimeStampCache defaultTimeStamp
             let projectTimeStamp = builder.GetLogicalTimeStampForProject(cache)
-            return PartialCheckResults (boundModel, timestamp, projectTimeStamp), ilAssemRef, tcAssemblyDataOpt, tcAssemblyExprOpt
+            let res = PartialCheckResults (boundModel, timestamp, projectTimeStamp), ilAssemRef, tcAssemblyDataOpt, tcAssemblyExprOpt
```

Comment:
Is this rearrangement necessary? just looking for minimal diffs.

Reply:
Sorry, this is leftover - at some point I needed `activity.Dispose()` before I extended the comp expression to support it implicitly. Will revert this line 👍